src/DataBuilder.php

```diff
@@ -775,10 +775,10 @@ public function scrub(&$data, $replacement = '*')
                 $parsedValues = array_values($parsed);
                 
                 /**
-                 * If we have at least one key/value pair (i.e. a=b) then
+                 * If we have at least two pairs (i.e. a=b&c=d) then
```

Comment:
I actually think looking at this more closely that we should get rid of this code.

We use `parse_url($data, PHP_URL_QUERY);` to attempt to get a query string, which will get the proper query string for these cases:
```
http://whatever.com/?a=b
whatever.com/?a=b
?a=b
```

If we have a bare string `a=b` then assuming that is a query string is a bit of stretch. I think we should just rely on parse_url to do it's job. The incidence of bare query strings without being part of some larger url must be very small.